### ISSUE TITLE
Release Process for v0.16.1

### ISSUE THREAD

**Issue by PastaPastaPasta** [dashpay/dash#3787]
Release Process

- [x]     Update translations, see translation_process.md. (not needed for v0.16.1)
- [x]     Update manpages, see gen-manpages.sh. #3798 
- [x]     Archive release notes for the old version to doc/release-notes/dash/, add release notes for the new version (in the release branch) #3805

Before every minor and major release:

- [x]     Update bips.md to account for changes since the last release. (not needed for v0.16.1)
- [x]     Update version in configure.ac (don't forget to set CLIENT_VERSION_IS_RELEASE to true)  (Merge directly to v0.16.x branch) #3788 
- [x]     Write release notes (Merge directly to v0.16.x branch) #3805
- [x]     Update src/chainparams.cpp nMinimumChainWork with information from the getblockchaininfo rpc. #3789 
- [x]     Update src/chainparams.cpp defaultAssumeValid with information from the getblockhash rpc. The selected value must not be orphaned so it may be useful to set the value two blocks back from the tip. Testnet should be set some tens of thousands back from the tip due to reorgs there. This update should be reviewed with a reindex-chainstate with assumevalid=0 to catch any defect that causes rejection of blocks in the past history. #3789 

Before every major release (all done since this isn't a major release):

- [x]     Update hardcoded seeds. TODO: Give example PR for Dash
- [x]     Update BLOCK_CHAIN_SIZE to the current size plus some overhead.
- [x]     Update src/chainparams.cpp chainTxData with statistics about the transaction count and rate. Use the output of the RPC getchaintxstats, see this pull request for an example. Reviewers can verify the results by running getchaintxstats <window_block_count> <window_last_block_hash> with the window_block_count and window_last_block_hash from your output.
- [x]     Update version of contrib/gitian-descriptors/*.yml: usually one'd want to do this on master after branching off the release - but be sure to at least do it before a new major release

Tag the Release

- [x]     Tag v0.16.1.0 and push

Do Gitian Build (for unsigned binaries) and create PRs

- [x]     Pasta Gitian Build
- [x]     Udjin Gitian Build
- [x]     Dustin Gitian Build
- [x]     Thephez Gitian Build

Create Windows/OS X detached signatures:
    Only once the Windows/OS X builds each have 3 matching signatures may they be signed with their respective release keys.

- [x]     Pasta creates windows detached signature and commits
- [x]     Dustin creates OS X detached signature

Update Gitian Builds with signed binaries

- [x]     Pasta Gitian Build
- [x]     Udjin Gitian Build
- [x]     Dustin Gitian Build
- [x]     Thephez Gitian Build

After 3 or more people have gitian-built and their results match:

- [x]     Create SHA256SUMS.asc for the builds
- [x]     GPG sign each download / binary
- [x]     Upload zips and installers, as well as SHA256SUMS.asc from last step, to github as github draft release. 
- [x]     Validate SHA256SUMS.asc and all binaries attached to github draft release are correct
- [x]     Notarize OS X binaries
- [x]     Update dash.org
- [x]     Release on github once dash.org is updated
- [x]     Fast-forward merge master to be in line with v0.16.x branch

Announce the release:

- [x]      Release on Dash forum: https://www.dash.org/forum/topic/official-announcements.54/
- [x]      Optionally Discord, twitter, reddit /r/Dashpay, ... but this will usually sort out itself
- [x]      Celebrate

Later:

- [x]     Merge back master into develop, see https://github.com/dashpay/dash/pull/3820

**Comment by xdustinface** [comment]
All done!
